stories/form.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+
+import Form from '../src/components/Form';
+
+const fieldset =  (
+  <fieldset className="well">
+    <legend>Some input fields</legend>
+    <input placeholder="A field"></input>
+    <input placeholder="Another field"></input>
+  </fieldset>
+);
+const form = storiesOf('Components/Form', module)
+  .add('default', () => {
+    return [
+      <Form
+      title="Default Form"
+      onSubmit={() => {}}
```

Comment:
You imported `@storybook/addon-actions` but are not using it. Here's is a good chance to use it: `onSubmit={action('clicked')}`.